hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/NFTOwnersChangeStep.java

```diff
@@ -123,7 +123,7 @@ public void doIn(final TransferContext transferContext) {
      * @param tokenId token id of the nft
      * @param nft nft to be transferred
      */
-    private void validateSpenderHasAllowance(
+    public static void validateSpenderHasAllowance(
```

Comment:
Does this need to be `public`? I thought package private would work
```suggestion
   static void validateSpenderHasAllowance(
```